lib/Widgets/Settings.vala

```diff
@@ -69,6 +73,36 @@ namespace Granite {
             }
         }
 
+        /**
+         * Possible clock format preferences expressed by the user
+         */
+        public enum ClockFormat {
```

Comment:
Reverse the order of the fields, we want 24H = 0 and 12H = 1 :slightly_smiling_face: